### DISCUSSION TITLE
Would it be more accurate to use the energy of each atom to calculate the force than to use the average of the total energy to calculate the force?

### DISCUSSION THREAD

**Discussion by yyyien** [deepmodeling/deepmd-kit#2016] [Q&A]
Hello!
Is it enough to calculate the force by using the average of the total enengy?

**Comment by njzjz** [comment]
I don't quite understand your meaning. The force is the negative gradients of the total energy:

$$
\mathbf F = - \nabla E
$$

i.e.

$$
\mathbf F_{ik} = -\frac{\partial{E}}{\partial \mathbf x_{ik}} \\
=- \sum_j  \frac{\partial{E_j}}{\partial\mathbf x_{ik}}
$$

**Reply by yyyien** [reply]
Does the energy output by deepmd-kit represent the energy of the whole system or the energy of individual atoms?

**Reply by njzjz** [reply]
The output energy is the system energy.

**Reply by yyyien** [reply]
I get it.Thank you!
